polaris-migrator/src/utilities/sass.ts

```diff
@@ -1,5 +1,18 @@
 import type {Node, ParsedValue, FunctionNode} from 'postcss-value-parser';
 
+/**
+ * Non-exhaustive CSS ident token RegExp:
+ * - https://drafts.csswg.org/css-syntax-3/#ident-token-diagram
+ * - https://www.w3.org/TR/CSS21/syndata.html#:~:text=%5B%2D%5D%3F%7Bnmstart%7D%7Bnmchar%7D*
+ */
+export const identTokenRegExp = /-?[_a-zA-Z][_a-zA-Z0-9-]*/;
```

Comment:
Nice RegExp utilities! Also nice reference links.

For my own understanding though, can you clarify or give an example of why this is needed/helpful for the namespace?

Reply:
Update: Discussed offline and removed. While it was valuable to understand how CSS/Sass identifiers are constructed, from a specification perspective, reimplementing them with RegExps adds little value to our migrations and could cause confusion for new comers to the platform.